operator/cluster/upgrade.go

```diff
@@ -129,7 +129,7 @@ func AddUpgrade(clientset *kubernetes.Clientset, restclient *rest.RESTClient, up
 	if err != nil {
 		log.Errorf("error submitting upgraded pgcluster CRD for cluster recreation of cluster %s, Error: %v", pgcluster.Name, err)
 	} else {
-		log.Debugf("upgraded cluster %s submitted for recreation, workflowid: %s", pgcluster.Name)
+		log.Debugf("upgraded cluster %s submitted for recreation", pgcluster.Name)
```

Comment:
s/recreation/re-creation

Reply:
This was part of the original description, not going to change it for now.